File: enigma-machine/javascript-version/keyboard.js
//keyboard
class Keyboard{
    forward(self, letter){
        signal = "ABCDEFGHIJKLMNOQRSTUVWXYZ".indexOf(letter)
        return signal
    }

    backward(self, signal){
        letter = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"[letter]
        return letter
    }
}

File: src/python/historical/enigma/keyboard.py
class Keyboard:
    
    def forward(self, letter):
        signal = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".find(letter)
        return signal
    
    def backward(self, signal):
        letter = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"[signal]
        return letter